lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -1691,20 +1691,23 @@ sub _carry_over_candidate ($self) {
     my $lookup_depth = $config->{lookup_depth};
     my $state_changes_limit = $config->{state_changes_limit};
 
+    my $label = sprintf "__carry_over_candidate(%d)", $self->id;
+    log_debug(sprintf "$label: _failure_reason=%s", $current_failure_reason);
```

Comment:
All the underscores look rather ugly for a log message. What about:

```suggestion
    my $label = sprintf "carry_over_candidate(%d)", $self->id;
    log_debug("$label: failure reason: $current_failure_reason");
```

Reply:
The double underscore is actually a typo, but the method is called `_carry_over_candidate`

Reply:
Keep in mind that these are debugging messages. I can just double click on `_failure_reason` in the code and paste it in the log view and search for it

Reply:
I suppose one `_` is *ok*.